REGEX_Vinbudin_kodi2.py

```diff
@@ -0,0 +1,84 @@
+import re
+import requests
+import pandas as pd
+from datetime import datetime
+import os
+
+# Load URLs from a GitHub file link
+def load_urls_from_github(github_url):
+    response = requests.get(github_url)
+    if response.status_code == 200:
+        urls = [line.strip() for line in response.text.splitlines() if line.strip() and not line.startswith('#')]
+        return urls
+    else:
+        print(f"Failed to retrieve URL list from {github_url}")
+        return []
+
+# Fetch the HTML from a URL
+def fetch_html(url):
+    response = requests.get(url)
+    if response.status_code == 200:
+        return response.text
+    else:
+        print(f"Failed to retrieve data from {url}")
+        return None
```

Comment:
Getið stytt og einfaldað þennan kóða töluvert. Hér er einföldun sem þið getur einfaldað fyrir ykkur (ég er búinn að prófa hana og hún virkar):

```suggestion
def load_urls(github_url):
    response = requests.get(github_url)
    return [line.strip() for line in response.text.splitlines() if line.strip() and not line.startswith('#')] if response.status_code == 200 else []

def fetch_html(url):
    response = requests.get(url)
    return response.text if response.status_code == 200 else None
```

Reply:
Okey, líklega geri það